src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs

```diff
@@ -101,6 +101,18 @@ public override void ExecuteCmdlet()
                     ||
                     Force.IsPresent)
```

Comment:
Force.IsPresent should not be part of this if.  The ShouldContinue should stand on its own